### PR TITLE
Rapidsai master add reading files and change training files

### PR DESCRIPTION
**Pull request by wangxiaoyunNV**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

File: examples/cugraph-pytorch/cugraph-local/data_loader/read_cora.py [hunk 1]
```diff
@@ -11,39 +11,57 @@
...
-    pg._edge_prop_dataframe.drop(columns=['0', '1'], inplace=True)
+    pg.add_edge_data(cora_M, vertex_col_names=("0","1"))
+    pg.add_vertex_data(cora_content1, vertex_col_name = "0")
+
+    pg._vertex_prop_dataframe.drop(columns = ['0'], inplace = True)
+    pg._edge_prop_dataframe.drop(columns = ['0', '1'], inplace = True)
 
-    # now load the wertex data, the last column (idx 1434) is the labels
-    cora_content = cudf.read_csv(feat_path, sep='\t', header=None)
     labels = cora_content['1434']
```
Comment (BradReesWork) [inline examples/cugraph-pytorch/cugraph-local/data_loader/read_cora.py]: Why not extract this column at line 35.  Then drop the column.
Currently you are maintaining two copies of the data

File: examples/cugraph-pytorch/cugraph-local/data_loader/read_reddit.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import random
+import sklearn
+import pandas as pd
+import scipy.sparse as sp
+import os
+
+
+def read_reddit(raw_path, self_loop=False):
+    #url = 'https://data.dgl.ai/dataset/reddit.zip'
+    #raw_path = "/home/xiaoyunw/Downloads/reddit"
```
Comment (BradReesWork) [inline examples/cugraph-pytorch/cugraph-local/data_loader/read_reddit.py]: you should not include local personal paths in code

File: examples/cugraph-pytorch/cugraph-local/data_loader/read_reddit.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    coo_adj = sp.load_npz(os.path.join(raw_path, "reddit_graph.npz"))
+    csr_adj = coo_adj.tocsr()
```
Comment (BradReesWork) [inline examples/cugraph-pytorch/cugraph-local/data_loader/read_reddit.py]: This is a very slow conversion.  You have the data in COO, why not just use that format? 

    edgelist = cudf.DataFrame()
    edgelist['src'] = cudf.Series(coo_adj.row)
    edgelist['dst'] = cudf.Series(coo_adj.col)
    edgelist['wt'] = cudf.Series(coo_adj.data)

This block is about 38x faster

Now the "tocsr" call does symmeterize the graph

File: examples/cugraph-pytorch/cugraph-local/data_loader/read_reddit.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    pg._vertex_prop_dataframe.drop(columns = ['name'], inplace = True)
+
+    gstore = CuGraphStorage(pg)
+
+    return gstore, labels, train_mask, val_mask, test_mask
+
+
+
+if __name__ == '__main__':
+    raw_path = "/home/xiaoyunw/Downloads/reddit"
```
Comment (BradReesWork) [inline examples/cugraph-pytorch/cugraph-local/data_loader/read_reddit.py]: remove reference to you personal directories

**Review by BradReesWork — CHANGES_REQUESTED** [review]
Please run flake8

File: examples/cugraph-pytorch/cugraph-local/data_loader/read_reddit.py (hunk 3 above)
Comment (wangxiaoyunNV) [reply examples/cugraph-pytorch/cugraph-local/data_loader/read_reddit.py]: changed, thank you for the advice.

File: examples/cugraph-pytorch/cugraph-local/data_loader/read_reddit.py (hunk 2 above)
Comment (wangxiaoyunNV) [reply examples/cugraph-pytorch/cugraph-local/data_loader/read_reddit.py]: just deleted the urls

File: examples/cugraph-pytorch/cugraph-local/data_loader/read_reddit.py (hunk 4 above)
Comment (wangxiaoyunNV) [reply examples/cugraph-pytorch/cugraph-local/data_loader/read_reddit.py]: I just deleted it.

**Pull request merged**